cli/src/main.rs

```diff
@@ -463,6 +483,43 @@ async fn run_cache(command: &CacheCommand) -> Result<(), CliError> {
     Ok(())
 }
 
+async fn run_init(args: &InitArgs) -> Result<(), CliError> {
+    match args.language.to_lowercase().trim() {
+        "rust" => {
+            println!(
+                "{:?}",
+                std::process::Command::new("git")
+                    .arg("clone")
+                    .arg(RUST_TEMPLATE)
+                    .output()
+            );
+        }
+        "c++" | "cpp" => {
+            println!(
+                "{:?}",
+                std::process::Command::new("git")
+                    .arg("clone")
+                    .arg(CPP_TEMPLATE)
+                    .output()
```

Comment:
(This comment didn't appear in my review for some reason)

`git clone`ing is smart here, but I think we should remove the `.git` directory since we don't really want our template being a git repo. We can do this in two ways:
* Keep it `git clone`, then `cd` into the dir and `rm -r .git`/`rmdir .git`
* Download it as a zip file and unzip it